akka-sample-sharding-scala/killrweather/src/main/resources/logback.xml

```diff
@@ -7,7 +7,7 @@
          logging config, see the Akka docs: https://doc.akka.io/docs/akka/2.6/typed/logging.html#logback -->
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>[%date{ISO8601}] [%level] [%logger] [%thread] [%X{akkaSource}] - %msg%n</pattern>
+            <pattern>[%date{ISO8601}] [%level] [%X{akkaAddress}] [%logger] [%thread] [%X{akkaSource}] - %msg%n</pattern>
```

Comment:
Include host:port in log makes it nicer to read

Reply:
Oh sweet.